src/App.js

```diff
@@ -152,12 +152,12 @@ export default function App() {
               />
 
               {/* /state/ routes are deprecated but still supported. */}
-              <Route exact path="/state/:stateId" component={LocationPage} />
-              <Route
-                exact
-                path="/state/:stateId/county/:countyId"
-                component={LocationPage}
-              />
+              <Route exact path="/state/:stateId">
+                <Redirect to="/us/:stateId" />
+              </Route>
+              <Route exact path="/state/:stateId/county/:countyId">
+                <Redirect to="/us/:stateId/county/:countyId" />
+              </Route>
```

Comment:
1. This seems unrelated to sitemap, so just want to confirm this is intentional.
2. Won't this redirect to our "old" URLs?  (i.e. `/us/wa` instead of `/us/washington-wa/`)?  Is that useful?

Reply:
One way to tell Google which page is the canonical is to redirect to the canonical version. I'm thinking of doing this in 2 steps:

1. redirect `state/wa` → `us/wa`
2. redirect `us/wa` → `us/washington-wa` (on a separate PR)

The second redirect is a bit more tricky, because we need to write the code to find the region by state code and redirect, the first one is straightforward. I can remove this change from this PR and implement both things on a separate PR if that's better!

Reply:
I implemented the redirect on a follow up PR https://github.com/covid-projections/covid-projections/pull/2279